cpp/bench/ann/src/common/ann_types.hpp

```diff
@@ -73,7 +76,7 @@ struct AlgoProperty {
 
 class AnnBase {
  public:
-  using index_type = size_t;
+  using index_type = uint32_t;
```

Comment:
Do we want to change this for all benchmarks?

Reply:
That was a temporary change for testing, I've removed that in cuVS PR (cuVS doesn't support uint32_t indexes in the refinement step)